eng/verify-nupkgs.ps1

```diff
@@ -89,6 +89,10 @@ function Verify-Nuget-Packages {
             if ($expectedNumOfFiles[$packageKey] -ne $actualNumOfFiles) {
                 $errors += "Number of files are not equal for '$packageBaseName', expected: $($expectedNumOfFiles[$packageKey]) actual: $actualNumOfFiles"
             }
+
+            if ($packageKey -eq "Microsoft.TestPlatform") {
+                Verify-Version -nugetDir $unzipNugetPackageDir
```

Comment:
Pass $errors here so you can append to them.

Reply:
Fixed.